wordfreq/__init__.py

```diff
@@ -328,24 +243,25 @@ def half_harmonic_mean(a, b):
 
 
 @lru_cache(maxsize=CACHE_SIZE)
-def word_frequency(word, lang, wordlist='combined', default=0.):
+def word_frequency(word, lang, wordlist='combined', minimum=0.):
```

Comment:
I think the point here was not just terminology: if we would return a value less than the minimum, we want to return the minimum instead.